cross-compiling/automatic_cross_compile.sh

```diff
@@ -22,43 +24,40 @@ source $THIS_DIR/env.sh $TARGET
 bash $THIS_DIR/get_sysroot.sh
 
 # Remove ROS2 old cross-compilation workspace and get a new one
-cd $BASE_DIR
-sudo rm -rf ros2_cc_ws
-mkdir -p ros2_cc_ws/src
-cd ros2_cc_ws
-wget https://raw.githubusercontent.com/ros2/ros2/"$ROS2_DISTRO"/ros2.repos
-vcs import src < ros2.repos
+sudo rm -rf $BASE_DIR/ros2_cc_ws
+bash $THIS_DIR/get_ros2_sources.sh --distro=$ROS2_DISTRO --ros2-path=$BASE_DIR/ros2_cc_ws
```

Comment:
Where is `BASE_DIR` defined?

Reply:
it's in line 18

Reply:
Ok, for some reason I completely missed it before...